common/src/lib.rs

```diff
@@ -527,12 +526,16 @@ pub fn wait_key(prog_id: H256, msg_id: H256) -> Vec<u8> {
     key
 }
 
-pub fn get_code(code_hash: H256) -> Option<Vec<u8>> {
+pub fn get_code(code_hash: H256) -> Option<CheckedCode> {
     sp_io::storage::get(&code_key(code_hash, CodeKeyPrefixKind::RawCode))
+        .map(|bytes| CheckedCode::decode(&mut &bytes[..]).expect("CheckedCode encoded correctly"))
```

Comment:
Stupid question, are we trying to avoid `expect`s, aren't we?

Reply:
Yeah, but this one can't be removed easily. Actually all other code in common uses the same approach and it is supposed that all modifications in the storage are done with the help of the API. So we almost sure this expect cannot panic.

Also we planned to refactor `common` create and move all storage-logic to pallets. Then these `expect`s should gone.

Reply:
Got it, thanks. I guess we may add `qed` for `expect`s that never occurs as we do in some places:

https://github.com/gear-tech/gear/blob/master/node/src/chain_spec.rs#L39